bql/planner/planner_test.go

```diff
@@ -797,6 +704,53 @@ func TestPlannerQuery(t *testing.T) {
 	}
 }
 
+func TestPlannerQueryError(t *testing.T) {
+	testTable := []struct {
+		q string
+	}{
```

Comment:
Question to you: is there something in the error you want to test? For example that you return the expected message, or that it has some other property (e.g. message contains something you're looking for, error is annotated with a particular status...)?
It's fine if you don't, but if you do this would be the place to put it :)

Reply:
Not for now, as I discussed here with Thiago, but we will keep an eye on this for the future. Thank you again for the suggestion Tati.